client/app/components/routes/select-operator/form/AccessRequestReceived.tsx

```diff
@@ -0,0 +1,44 @@
+import Link from "next/link";
+import { BC_GOV_LINKS_COLOR } from "@/app/styles/colors";
+import { Operator } from "@/app/components/routes/select-operator/form/types";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import { getOperator } from "@/app/components/routes/select-operator/form/ConfirmSelectedOperator";
+
+export default async function AccessRequestReceived({
+  params,
+}: {
+  params: { id: number };
+}) {
+  const operator: Operator | { error: string } = await getOperator(params.id);
+
+  if ("error" in operator) {
```

Comment:
Because the operator variable can be either an Operator or an object with an `error` property, I ensure that if `fetchApi` fails and returns an error, we do not return the rest of the component.